examples/enitity/webapp/src/index.css

```
@@ -0,0 +1,18 @@
+@import "tailwindcss";
+@import "@flanksource/clicky-ui/styles.css";
+@source "../node_modules/@flanksource/clicky-ui/dist";
```

Stylelint (17.7.0)

[error] Unexpected unknown at-rule "@source" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)